release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -68,6 +68,8 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
 
+==== New minimum system requirements for installing {product-title} on a single node
+This release updates the minimum system requirements for installing {product-title} on a single node. When installing {product-title} on a single node, you should configure a minimum of 16 GB of RAM. Specific workload requirements can require additional RAM. The complete list of supported platforms has been updated to include bare metal, vSphere, {rh-openstack-first}, and Red Hat Virtualization platforms. In all cases, you must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file in a case that `openshift-installer` binary is being used to install single node {product-title}.
```

Comment:
I think there is an attribute for SNO now: 

:sno: single-node OpenShift

:sno-caps: Single-node OpenShift

You might use it in some cases here (like the last mentioning of single node {product-title}.

Reply:
Updated as suggested.